tutorials/clinical/10_ieeg_localize.py

```diff
@@ -289,6 +289,30 @@ def plot_overlay(image, compare, title, thresh=None):
 # after closing the GUI and can now be saved
 gui.close()  # close when done
 
+# %%
+# Let's do a quick sidebar and show what this looks like for ECoG as well.
+
+T1_ecog = nib.load(op.join(misc_path, 'ecog', 'sample_ecog', 'mri', 'T1.mgz'))
+CT_orig_ecog = nib.load(op.join(misc_path, 'ecog', 'sample_ecog_CT.mgz'))
+
+# pre-computed affine from `mne.transforms.compute_volume_registration`
+reg_affine = np.array([
+    [0.99982382, -0.00414586, -0.01830679, 0.15413965],
+    [0.00549597, 0.99721885, 0.07432601, -1.54316131],
+    [0.01794773, -0.07441352, 0.99706595, -1.84162514],
+    [0., 0., 0., 1.]])
+# align CT
+CT_aligned_ecog = mne.transforms.apply_volume_registration(
+    CT_orig_ecog, T1_ecog, reg_affine)
+
+raw_ecog = mne.io.read_raw(op.join(misc_path, 'ecog', 'sample_ecog_ieeg.fif'))
+# use estimated `trans` which was used when the locations were found previously
+subj_trans_ecog = mne.coreg.estimate_head_mri_t(
+    'sample_ecog', op.join(misc_path, 'ecog'))
+gui = mne.gui.locate_ieeg(raw_ecog.info, subj_trans_ecog, CT_aligned_ecog,
+                          subject='sample_ecog',
+                          subjects_dir=op.join(misc_path, 'ecog'))
```

Comment:
@alexrockhill it would be awesome if we could do here the same thing we do for ReportScraper and BrainScraper. I think it will only require ~20 lines and adapting https://doc.qt.io/qt-5/qtwidgets-desktop-screenshot-example.html. Do you want to look in this PR? It would be a nice sanity check, and I don't think there's a rush to get this PR in...

Reply:
... some variant of `screen.grabWindow( widget.winId())` should get you the image to embed I think

Reply:
Looks like the current `dev` is broken https://mne.tools/dev/auto_examples/visualization/3d_to_2d.html so it might be good to do this as a follow up

Reply:
Your comment links to the `3d_to_2d` example, but I see what you mean with https://mne.tools/dev/auto_tutorials/clinical/10_ieeg_localize.html#marking-the-location-of-each-electrode-contact as well. Okay for me for a follow-up PR, do you want to tackle it?

Reply:
You're welcome to do a follow-up PR or push to this one, I didn't realize that the scrapper was causing such plotting issues. I linked to the example because it's all messed up on the current development branch as it uses a mix of new and old (someone already updated the misc version).